draft-ietf-quic-http.md

```diff
@@ -811,14 +926,17 @@ PRIORITY (0x2):
 
 RST_STREAM (0x3):
 : RST_STREAM frames do not exist, since QUIC provides stream lifecycle
-  management.
+  management.  The same code point is used for the CANCEL_PUSH frame
+  ({{frame-cancel-push}}).
 
 SETTINGS (0x4):
 : SETTINGS frames are sent only at the beginning of the connection.  See
   {{frame-settings}} and {{h2-settings}}.
 
 PUSH_PROMISE (0x5):
-: See {{frame-push-promise}}.
+: The PUSH_PROMISE does not reference the stream that carries the response to
+  the pushed request; instead the push stream references the PUSH_PROMISE frame.
```

Comment:
reword. suggestion: "The PUSH_PROMISE does not reference the stream carrying the promised resource; instead the push stream references the PUSH_PROMISE frame."